tests/runtime/events/test_events_producers.py

```diff
@@ -6,38 +6,6 @@
 )
 
 
-def test_event_producer_produce_events(
```

Comment:
Removing this test, just because it failing without any issue/stacktrace and tests keep been green